.github/workflows/ca-publishing-crl-file-test.yml

```diff
@@ -166,82 +282,350 @@ jobs:
+          docker exec pki find /var/lib/pki/pki-tomcat/crl -name "MasterCRL-*.der" | tee output
 
-          # there should be one CRL file
+          # there should be one timestamped CRL file
           cat output | wc -l > actual
           echo "1" > expected
           diff expected actual
 
-          FILENAME=$(cat output)
-          echo "FILENAME: $FILENAME"
-
           # check the latest CRL
           docker exec pki openssl crl \
-              -in $FILENAME \
+              -in /var/lib/pki/pki-tomcat/crl/MasterCRL.crl \
               -inform DER \
               -text \
               -noout | tee output
 
-          # there should be no certs in the latest CRL
+          # CRL should contain no certs
           sed -n "s/^\s*\(Serial Number:.*\)\s*$/\1/p" output | wc -l > actual
           echo "0" > expected
           diff expected actual
 
-      - name: Check CRL after cert revocation
+      - name: Check user cert after update
+        run: |
+          # check user cert using OpenSSL
+          docker exec pki openssl verify \
+              -crl_check \
+              -crl_download \
+              -CAfile ca_signing.crt \
+              testuser.crt \
+              > >(tee stdout) 2> >(tee stderr >&2) || true
+
+          # user cert should be valid
+          echo "testuser.crt: OK" > expected
+          diff expected stdout
+
+          # check user cert using NSS with PKIX
+          docker exec pki /usr/lib64/nss/unsupported-tools/vfychain \
+              -d /root/.dogtag/nssdb \
+              -a \
+              -u 0 \
+              -pp \
+              -g leaf \
+              -m crl \
+              testuser.crt \
+              > >(tee stdout) 2> >(tee stderr >&2) || true
+
+          # user cert should be valid
+          echo "Chain is good!" > expected
+          diff expected stderr
+
+      - name: Check server cert after update
+        run: |
+          # download the latest CRL
+          docker exec pki curl -sJO http://pki.example.com:8080/crl/MasterCRL.crl
+
+          # convert CRL to PEM
+          docker exec pki openssl crl \
+              -in MasterCRL.crl \
+              -inform DER \
+              -out MasterCRL.pem \
+              -outform PEM
+
+          # check server cert using OpenSSL
+          docker exec pki openssl verify \
+              -crl_check \
+              -CRLfile MasterCRL.pem \
+              -CAfile ca_signing.crt \
+              test.example.com.crt \
+              > >(tee stdout) 2> >(tee stderr >&2) || true
+
+          # server cert should be valid
+          echo "test.example.com.crt: OK" > expected
+          diff expected stdout
+
+          # import CRL into NSS
+          docker exec pki crlutil -I -d /root/.dogtag/nssdb -i MasterCRL.crl
+          docker exec pki crlutil -L -d /root/.dogtag/nssdb -n ca_signing
+
+          # check server cert using NSS without PKIX
```

Comment:
In this case is not used PKIX but the legacy verification method

Reply:
I removed references to PKIX in all comments to avoid confusions.